### PR TITLE
Add Nonproliferation Sanctions (ISN) data to Watchman

### PR DESCRIPTION
**Pull request by ckelly-digicert**
Added support for parsing and searching ISN data within Watchman.

### CODE REVIEW

**Comment by ckelly-digicert** [comment]
Note that for the ISN data, the `Remarks` and `AdditionalNames` fields are optional. As such, the current expansion helper function ends up setting the values with the resulting JSON document as `null`.

**Comment by ckelly-digicert** [comment]
@adamdecaf It would appear that a recent change from `origin/master` may have broken a unit test unrelated to the changes in this PR.

**Comment by adamdecaf** [comment]
> @adamdecaf It would appear that a recent change from `origin/master` may have broken a unit test unrelated to the changes in this PR.

Those Watch tests are a bit flakey. I re-ran it and the test passed.

**Comment by adamdecaf** [comment]
I'm not seeing /v1/ endpoints, but Docker Hub is adjusting some stuff. That could be the cause of these build failures.

Screen Shot 2022-08-29 at 12 31 32 PM

**Comment by adamdecaf** [comment]
Yep. I've tweaked that logging logic [so `master` should be fixed now](https://github.com/moov-io/watchman/commit/5602c39960dfac1d034812705cf3523687a6b602).

File: cmd/server/download.go [hunk 1]
```diff
@@ -59,10 +59,11 @@ type DownloadStats struct {
...
 	// Consolidated Screening List (CSL)
-	BISEntities       int `json:"bisEntities"`
-	MilitaryEndUsers  int `json:"militaryEndUsers"`
-	SectoralSanctions int `json:"sectoralSanctions"`
-	Unverified        int `json:"unverifiedCSL"`
+	BISEntities               int `json:"bisEntities"`
+	MilitaryEndUsers          int `json:"militaryEndUsers"`
+	SectoralSanctions         int `json:"sectoralSanctions"`
+	Unverified                int `json:"unverifiedCSL"`
+	NonproliferationSanctions int `json:"nonproliferationSanctions"`
```
Comment (adamdecaf) [inline cmd/server/download.go:66]:
```suggestion
	NonProliferationSanctions int `json:"nonProliferationSanctions"`
```

File: cmd/server/search_us_csl.go [hunk 2]
```diff
@@ -93,3 +93,14 @@ func (s *searcher) TopUVLs(limit int, minMatch float64, name string) []*Result[c
 
 	return topResults[csl.UVL](limit, minMatch, name, s.UVLs)
 }
+
+// TopISNs searches Nonproliferation Sanctions records by Name and Alias
+func (s *searcher) TopISNs(limit int, minMatch float64, name string) []*Result[csl.ISN] {
```
Comment (adamdecaf) [inline cmd/server/search_us_csl.go:98]: I'm going to add a test for this, but the change looks good overall.

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**